nkms/network/upgrade.py

```diff
@@ -0,0 +1,30 @@
+import os
+import sys
+
+
+def ssl_config(server):
+    import ssl
+
+    cafile = capath = None
+
+    context = ssl.create_default_context(
+        ssl.Purpose.CLIENT_AUTH, cafile=cafile, capath=capath)
+
+    context.load_default_certs()
+    context.load_cert_chain("/home/hubcraft-thinkpad/git/nucypher-kms/_temp/server.pem", password="nucypher")
```

Comment:
What's wrong with storing test certificates with the tests and getting a relative path?

Reply:
Haha right.  That's a mistake.  I'll tear that out too.  Actually I'm going to withdraw upgrade.py completely for the moment.

Reply:
Ah, right, a separate PR?

Reply:
Yeah.